habitat_baselines/common/utils.py

```diff
@@ -148,11 +99,7 @@ def batch_obs(
             batch[sensor].append(_to_tensor(obs[sensor]))
 
     for sensor in batch:
-        batch[sensor] = (
```

Comment:
cc @erikwijmans, is this change safe?

Reply:
@mathfac All tests pass so presumably and any RGB input is already converted implicitly to float in the encoders.

Reply:
Yeah, it should be fine